app/controllers/api/v1/users_controller.rb

```diff
@@ -12,6 +12,17 @@ def create
     end
   end
 
+  def update
+    user = User.update(params[:id], user_params)
+    if user.valid?
+      render json: UserSerializer.new(user), status: 200
+    else
+      render json: {
+        error: 'User name, email, birthdate cannot be empty'
```

Comment:
This is awesome! We can also do our part to prevent them from submitting an incomplete form on the FE.